server/@types/viewModels/index.d.ts

```diff
@@ -128,4 +128,28 @@ declare module 'viewModels' {
   // TODO RR-115 - define the fields
   // eslint-disable-next-line @typescript-eslint/no-empty-interface
   export interface SkillsAndInterests {}
+
+  export interface PrePrisonQualifications {
+    problemRetrievingData: boolean
+    highestEducationLevel?: string
+    targetDateRange?:
```

Comment:
targetDateRange ??

Reply:
Oops, you'd think I copied & pasted this from somewhere! 😂